### PR TITLE
ses/test - add test for prepareStackTrace override

### PR DESCRIPTION
**Pull request by kumavis**
this tests the behavior expected in the `depd` package

### CODE REVIEW

**Comment by kumavis** [comment]
test failure is very mysterious for me

**Comment by kriskowal** [comment]
This change applied on trunk passes tests. This may have been addressed by @warner’s change https://github.com/Agoric/SES-shim/pull/312.

File: packages/ses/test/tame-global-error-object-allow.test.js [hunk 1]
```diff
@@ -17,6 +17,15 @@ test('tameGlobalErrorObject', t => {
     t.equal(typeof error.stack, 'string');
     Error.captureStackTrace(error);
     t.equal(typeof error.stack, 'string');
+    // temporary Error.prepareStackTrace override
+    // as seen in "depd" package
+    const errObj = {};
```
Comment (kriskowal) [inline packages/ses/test/tame-global-error-object-allow.test.js:22]: In the separate case, this can be called `error`.

**Comment by erights** [comment]
Hi @kumavis I'm working on a PR that should subsume this PR and fix the incompatibility. I'll add you as a reviewer when it is ready.

On the way to fixing it, I discovered a v8 bug I just reported at https://bugs.chromium.org/p/v8/issues/detail?id=10551

**Comment by erights** [comment]
That v8 bug will be irritating to work around, but is not a show stopper.

**Comment by kumavis** [comment]
theres some notes on where else this pattern is used https://github.com/Agoric/SES-shim/issues/251

**Comment by erights** [comment]
Subsumed by #359 . Closing.

**Pull request closed without merging**